bufferevent-internal.h

```diff
@@ -306,11 +306,18 @@ extern const struct bufferevent_ops bufferevent_ops_pair;
 #define BEV_IS_FILTER(bevp) ((bevp)->be_ops == &bufferevent_ops_filter)
 #define BEV_IS_PAIR(bevp) ((bevp)->be_ops == &bufferevent_ops_pair)
 
-#if defined(EVENT__HAVE_OPENSSL)
-extern const struct bufferevent_ops bufferevent_ops_openssl;
-#define BEV_IS_OPENSSL(bevp) ((bevp)->be_ops == &bufferevent_ops_openssl)
+#if defined(EVENT__HAVE_OPENSSL) | defined(EVENT__HAVE_MBEDTLS)
+extern const struct bufferevent_ops bufferevent_ops_ssl;
+#define BEV_IS_SSL(bevp) ((bevp)->be_ops == &bufferevent_ops_ssl)
```

Comment:
But this has more problems then just a typo, same symbol in multiple libraries:

```
$ nm lib/libevent_openssl.so | fgrep ops_ssl
0000000000009d60 d bufferevent_ops_ssl
$ nm lib/libevent_mbedtls.so | fgrep ops_ssl
0000000000008d40 d bufferevent_ops_ssl
```

And this means that you can't use openssl with mbedtls